charts/vela-core/templates/definitions/autodetect.yaml

```diff
@@ -0,0 +1,8 @@
+apiVersion: core.oam.dev/v1beta1
+kind: WorkloadDefinition
+metadata:
+  name: autodetect.core.oam.dev
```

Comment:
Add an annotation description might help to explain the definition to those who are interested in it.

Reply:
got it